BatchRenderer/Mappings/ResolveMappingsValues.m

```diff
@@ -103,7 +103,19 @@
             map.right.value = fileInfo.resolvedPath;
             
             if ~fileInfo.isRootFolderMatch
```

Comment:
Nice.  I like what you did here.  I think copying files to the resource folder could be useful in other cases as well.  Would you be interested in making this a separate behavior, independent of the renderer or Docker?

We could define a separate hint like `copyResources`.  Then it could be
```
if hints.copyResources
```
instead of
```
if(hints.dockerFlag == 0 || strcmp(hints.renderer,'Mitsuba'))
```

A downside would be that docker users would have to remember to set the value of this new flag.  But I think it would be worth it to gain a nice, modular behavior.